pkg/client/oci/oci.go

```diff
@@ -62,18 +72,75 @@ func New(repo *api.Repo, c cache.Cacher, insecure bool) (*Repo, error) {
 	if err != nil {
 		return nil, errors.Trace(err)
 	}
+	resolver := newDockerResolver(u, repo.GetAuth().GetUsername(), repo.GetAuth().GetPassword(), insecure)
+	// Init entries
+	entries := make(map[string][]string)
```

Comment:
Initialize these variables closer to the context where they are used for the first time